### ISSUE TITLE
Font Optimization (Next.js) error

### ISSUE THREAD

**Issue by pontusab** [tamagui/tamagui#148]
We would like to use next.js Font Optimization still but having the stylesheet linked in <Head> gives us this error:
<img width="971" alt="Screenshot 2022-07-28 at 18 04 09" src="https://user-images.githubusercontent.com/655158/181585259-1dd3271a-f4b4-4cfe-93cb-202fc317f480.png">

**To Reproduce**
Add this to your `_document.tsx`:

```
<Head>
  <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
  <link href="https://fonts.googleapis.com/css2?family=Inter&display=optional" rel="stylesheet" />
</Head>
```

**System Info:** 
  System:
    OS: macOS 12.0.1
    CPU: (8) x64 Apple M1
    Memory: 52.30 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.15.1 - ~/.nvm/versions/node/v16.15.1/bin/node
    Yarn: 4.0.0-rc.13 - ~/.nvm/versions/node/v16.15.1/bin/yarn
    npm: 8.11.0 - ~/.nvm/versions/node/v16.15.1/bin/npm
    Watchman: 2022.07.04.00 - /usr/local/bin/watchman
  Browsers:
    Chrome: 103.0.5060.134
    Safari: 15.1

**Comment by natew** [comment]
Fix in https://github.com/tamagui/tamagui/pull/156/commits/0e9f02ed314ad29a34d2d271c0ef12166019b8d3

**Comment by natew** [comment]
Check for fix in beta.134

**Comment by pontusab** [comment]
Unfortunately that did not do the trick, here is my `_document.tsx` file:

```
// @ts-nocheck
import NextDocument, { Head, Html, Main, NextScript } from 'next/document';
import { Children } from 'react';
import { AppRegistry } from 'react-native';
import PermanentUI from '@permanent/ui';

export default class Document extends NextDocument {
  static async getInitialProps({ renderPage }: any) {
    AppRegistry.registerComponent('Main', () => Main);
    const page = await renderPage();

    const { getStyleElement } = AppRegistry.getApplication('Main');
    const styles = [
      getStyleElement(),
      <style key="permanent-ui-css" dangerouslySetInnerHTML={{ __html: PermanentUI.getCSS() }} />,
    ];
    return { ...page, styles: Children.toArray(styles) };
  }

  render() {
    return (
      <Html>
        <Head>
          <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
          {/* Note: https://github.com/tamagui/tamagui/issues/148 */}
          <link href="https://fonts.googleapis.com/css2?family=Inter&display=optional" rel="stylesheet" />
        </Head>
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    );
  }
}

```

**Comment by natew** [comment]
Alright let's give it another go, I think related to cross origin sheets. Fix here: 88621c320ea2157625eecf190d6ec962a77a45a3

**Comment by pontusab** [comment]
Can confirm it works, thank you! 💯
